galata/test/documentation/debugger.test.ts

```diff
@@ -160,6 +160,8 @@ test.describe('Debugger', () => {
 
     // Wait to be stopped on the breakpoint
     await page.debugger.waitForCallStack();
+    // Wait for the red debug indicator box to appear
+    await page.waitForTimeout(50);
```

Comment:
In general adding or increasing `page.waitForTimeout` will not help us much, except it will be less flaky. Instead, we need to find out what selector we want to await and add an await for that selector.